### ISSUE TITLE
[meater] Is cook ID useful?

### ISSUE THREAD

**Issue by jlaur** [openhab/openhab-addons#13943]
@jannegpriv - I was wondering if cook ID is actually useful?

https://github.com/openhab/openhab-addons/blob/831443334e72ce1f12d517f0816ca8a165dfe8f6/bundles/org.openhab.binding.meater/src/main/resources/OH-INF/thing/thing-types.xml#L143-L148

Very minor, but we could consider either removing it or making the channel advanced, so it would normally be hidden for users. It seems this is an ID used by the API without any real user value?

**Comment by jlaur** [comment]
@jannegpriv - in case you missed this, any comments?

**Comment by jannegpriv** [comment]
I agree that it is probably very value in that channel, so we can make it advanced.
I can make a PR for fixing that, need to fix a working 4.0 dev-environment or do you also want it fixed in 3.4.x?

**Comment by jlaur** [comment]
> I agree that it is probably very small value in that channel, so we can make it advanced.

I'm still curious, though: What are you using it for? 🙂 The other suggestion would be to completely remove it, but if there is a valid use case for it, it should rather be marked as advanced.

> I can make a PR for fixing that, need to fix a working 4.0 dev-environment or do you also want it fixed in 3.4.x?

No, this is not a bug, only a minor detail. Also there is no rush, just wondered if we are exposing internal information for no good reason.

Off-topic for this issue, but now that I got you. 😉 What would you think about decreasing the polling frequency when no cooks are active? The drawback would of course be it would take additional time to detect when a cook is started. And the advantage would be that we are making fewer calls when no cooks are active, which is probably most of the time. Personally I can have a month when I'm not using it at all, and if I remember correctly, the API is always polled every 30 seconds? If you agree this could be optimized, I'll create a separate issue.
